### PR TITLE
[RFR] Image embedding rgb and url support

### PR DESCRIPTION
**Pull request by rguo123**
Option to pass static URLs + rgb for jpg conversion

### CODE REVIEW

**Comment by rguo123** [comment]
@zanussbaum think this is good to go!

ran
image(images=[image_url], model="nomic-embed-vision-v1.5") on a static url of a sushi image

and got this:
https://atlas.nomic.ai/data/nomic-multimodal-series/cc3m-100k-image-bytes/map/144e9b38-97b7-4c6f-9de3-8eeac2c5e731#epGf

File: setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 setup(
     name="nomic",
-    version="3.0.29",
+    version="3.0.30",
```
Comment (zanussbaum) [inline setup.py:11]: i think this needs to be 3.0.31, can you merge from main?

**Review by zanussbaum — APPROVED** [review]
LGTM, just make sure you merge from main. I think the version needs to be bumped one more

File: setup.py (hunk 1 above)
Comment (rguo123) [reply setup.py]: ah thanks for the catch, rebased and bumped

**Pull request merged**